app/assets/stylesheets/application.scss

```diff
@@ -10,3 +10,7 @@
 @import "components/index";
 @import "pages/index";
 @import "plugins/index";
+
+html {
+  scroll-behavior: smooth;
```

Comment:
Nice! I learned about it only a few months ago. I'd probably have forgotten it and used the old way of doing this with JS stuff.